### PR TITLE
fix issue 20067 - [REG2.086] foreach no longer works on range with alias front

### PR DESCRIPTION
**Pull request by ghost**
Trivial, just a call to  `toAlias()` missing

### CODE REVIEW

File: test/compilable/b20067.d [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    int i;
+    @property int ii(){return 0;}
+    @property bool empty(){return true;}
+    alias front = ii;
+    void popFront(){}
+}
+struct S2
+{
+    int i;
+    @property bool empty(){return true;}
```
Comment (ZombineDev) [inline test/compilable/b20067.d]: Can you add/modify a test to include an alias also for `empty` for completeness?

**Review by PetarKirov — APPROVED** [review]

File: test/compilable/b20067.d (hunk 1 above)
Comment (Basile-z) [reply test/compilable/b20067.d]: done

Comment (ZombineDev) [reply test/compilable/b20067.d]: Thanks!

**Comment by kinke** [comment]
I guess the other cases (`sfront.isFuncDeclaration()` etc.) should be alias-resolved too.

**Comment by Basile-z** [comment]
@kinke, the test case shows that this is not required.

**Comment by Basile-z** [comment]
Good to merge then ?

**Comment by Basile-z** [comment]
ping @thewilsonator

**Comment by Basile-z** [comment]
ping @thewilsonator

**Comment by Basile-z** [comment]
one thing after contacting andrei that is clear, is that a bug fixer like me will get nothing, not a single fucking euro, so at least I expect a bit of respect when I fix a bug. When it's good, you merge.

**Comment by thewilsonator** [comment]
Sorry your ping came a 1am and 2am my time.
